### DISCUSSION TITLE
How can I access custom errors included in the `cause` of `TRPCError` on the client side?

### DISCUSSION THREAD

**Discussion by lucyj204** [trpc/trpc#5519] [Q&A] [answered]
I am throwing a `TRPCError` which includes a custom error in the `cause` and want to be able to access this custom error on the client side but it appears that the `cause` property in `data` from the `TRPCClientError` is missing. What is the best way to go about including custom error details in a `TRPCError` so that I can see the custom error details as part of the `TRPCClientError`, on the client side?

**Comment by KATT** [comment, marked as answer]
https://trpc.io/docs/server/error-formatting

**Reply by lucyj204** [reply]
Thank you 👍
